File: CodeSamples/python/swagger_client/models/business.py
# coding: utf-8

class Business(object):
    swagger_types = {
        'address': 'Address',
        'name': 'str',
        'phone': 'str',
        'fax': 'str',
        'web_site': 'str'
    }

    attribute_map = {
        'address': 'Address',
        'name': 'Name',
        'phone': 'Phone',
        'fax': 'Fax',
        'web_site': 'WebSite'
    }

    def __init__(self, address=None, name=None, phone=None, fax=None, web_site=None):  # noqa: E501
        """Business - a model defined in Swagger"""  # noqa: E501
        self._address = None
        self._name = None
        self._phone = None
        self._fax = None
        self._web_site = None
        self.discriminator = None
        if address is not None:
            self.address = address
        if name is not None:
            self.name = name
        if phone is not None:
            self.phone = phone
        if fax is not None:
            self.fax = fax
        if web_site is not None:
            self.web_site = web_site

    @property
    def address(self):
        return self._address

    @address.setter
    def address(self, address):
        self._address = address

    @property
    def name(self):
        return self._name

    @name.setter
    def name(self, name):
        self._name = name

    @property
    def phone(self):
        return self._phone

    @phone.setter
    def phone(self, phone):
        self._phone = phone

    @property
    def fax(self):
        return self._fax

    @fax.setter
    def fax(self, fax):
        self._fax = fax

    @property
    def web_site(self):
        return self._web_site

    @web_site.setter
    def web_site(self, web_site):
        self._web_site = web_site

File: CodeSamples/php/SwaggerClient-php/lib/Model/Business.php
<?php

namespace Swagger\Client\Model;

use \ArrayAccess;
use \Swagger\Client\ObjectSerializer;

class Business implements ModelInterface, ArrayAccess
{
    const DISCRIMINATOR = null;

    protected static $swaggerModelName = 'Business';

    protected static $swaggerTypes = [
        'address' => '\Swagger\Client\Model\Address',
        'name' => 'string',
        'phone' => 'string',
        'fax' => 'string',
        'web_site' => 'string'
    ];

    protected static $swaggerFormats = [
        'address' => null,
        'name' => null,
        'phone' => null,
        'fax' => null,
        'web_site' => null
    ];

    public static function swaggerTypes()
    {
        return self::$swaggerTypes;
    }

    public static function swaggerFormats()
    {
        return self::$swaggerFormats;
    }

    protected static $attributeMap = [
        'address' => 'Address',
        'name' => 'Name',
        'phone' => 'Phone',
        'fax' => 'Fax',
        'web_site' => 'WebSite'
    ];

    protected static $setters = [
        'address' => 'setAddress',
        'name' => 'setName',
        'phone' => 'setPhone',
        'fax' => 'setFax',
        'web_site' => 'setWebSite'
    ];

    protected static $getters = [
        'address' => 'getAddress',
        'name' => 'getName',
        'phone' => 'getPhone',
        'fax' => 'getFax',
        'web_site' => 'getWebSite'
    ];

    public static function attributeMap()
    {
        return self::$attributeMap;
    }

    public static function setters()
    {
        return self::$setters;
    }

    public static function getters()
    {
        return self::$getters;
    }

    public function getModelName()
    {
        return self::$swaggerModelName;
    }

    protected $container = [];

    public function __construct(array $data = null)
    {
        $this->container['address'] = isset($data['address']) ? $data['address'] : null;
        $this->container['name'] = isset($data['name']) ? $data['name'] : null;
        $this->container['phone'] = isset($data['phone']) ? $data['phone'] : null;
        $this->container['fax'] = isset($data['fax']) ? $data['fax'] : null;
        $this->container['web_site'] = isset($data['web_site']) ? $data['web_site'] : null;
    }

    public function listInvalidProperties()
    {
        $invalidProperties = [];

        return $invalidProperties;
    }

    public function valid()
    {
        return count($this->listInvalidProperties()) === 0;
    }

    public function getAddress()
    {
        return $this->container['address'];
    }

    public function setAddress($address)
    {
        $this->container['address'] = $address;

        return $this;
    }

    public function getName()
    {
        return $this->container['name'];
    }

    public function setName($name)
    {
        $this->container['name'] = $name;

        return $this;
    }

    public function getPhone()
    {
        return $this->container['phone'];
    }

    public function setPhone($phone)
    {
        $this->container['phone'] = $phone;

        return $this;
    }

    public function getFax()
    {
        return $this->container['fax'];
    }

    public function setFax($fax)
    {
        $this->container['fax'] = $fax;

        return $this;
    }

    public function getWebSite()
    {
        return $this->container['web_site'];
    }

    public function setWebSite($web_site)
    {
        $this->container['web_site'] = $web_site;

        return $this;
    }
    public function offsetExists($offset)
    {
        return isset($this->container[$offset]);
    }

    public function offsetGet($offset)
    {
        return isset($this->container[$offset]) ? $this->container[$offset] : null;
    }

    public function offsetSet($offset, $value)
    {
        if (is_null($offset)) {
            $this->container[] = $value;
        } else {
            $this->container[$offset] = $value;
        }
    }

    public function offsetUnset($offset)
    {
        unset($this->container[$offset]);
    }

    public function __toString()
    {
        if (defined('JSON_PRETTY_PRINT')) { // use JSON pretty print
            return json_encode(
                ObjectSerializer::sanitizeForSerialization($this),
                JSON_PRETTY_PRINT
            );
        }

        return json_encode(ObjectSerializer::sanitizeForSerialization($this));
    }
}
